helpers/wob.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Load the user's home wob configuration, if found.
+# Otherwise default to the openSUSEway configuration.
+
+set -uo pipefail
+
+WOB_FIFO="$SWAYSOCK.wob"
+WOB_HOME_CONFIG="$HOME/.config/wob/wob.ini"
+WOB_OPENSUSEWAY_CONFIG="/etc/sway/wob/wob.ini"
+
+################################################################
+
+# Kill any previous wob listener
+pkill -f "tail -f $WOB_FIFO"
```

Comment:
hm. so anyone can actually write in their sway .config file something like this:

exec pkill -f "tail -f $WOB_FIFO"
tail -f "$WOB_FIFO" | wob --config $HOME/.config/wob/wob.ini